### PR TITLE
Use pod monitor instead of service for metrics collection

### PR DESCRIPTION
**Pull request by uvegla**
## Checklist

- [ ] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Review by MarcelMue — APPROVED** [review]
LGTM but the monitoring monitorers should be able to verify

**Comment by uvegla** [comment]
> LGTM but the monitoring monitorers should be able to verify

Yep, that is what we agreed upon with @QuentinBisson.

**Comment by QuentinBisson** [comment]
Looks good but we need to do a change to prometheus-meta-operator first to uspport pod monitors
cc @weatherhog

File: helm/app-operator/templates/pod-monitor.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  namespace: {{ include "resource.default.namespace" . }}
+  labels:
+    name: {{ include "resource.default.name" . }}
+spec:
+  namespaceSelector:
+    matchNames:
+      - {{ include "resource.default.namespace" . }}
+  selector:
+    matchLabels:
+      {{- include "labels.common" . | nindent 6 }}
```
Comment (QuentinBisson) [inline helm/app-operator/templates/pod-monitor.yaml:14]:
```suggestion
      {{- include "labels.selector" . | nindent 6 }}
```

**Comment by QuentinBisson** [comment]
@uvegla I just released the new PMO with pod monitor support. Can you test your PR there ?

**Comment by MarcelMue** [comment]
This looks solid to me and CI is green - how is testing going?

**Comment by uvegla** [comment]
> This looks solid to me and CI is green - how is testing going?

Working on it on ginger. Struggling with Flux and deploying my change for testing at the moment.

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,16 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+## [6.0.0]
```
Comment (MarcelMue) [inline CHANGELOG.md:10]: Line should be removed?

**Pull request merged**